.github/workflows/release-java.yaml

```diff
@@ -0,0 +1,46 @@
+name: Build release
+
+# This is triggered by merging a PR from a branch named release-*. This workflow builds and publishes the Java release artifacts
+# to Maven Central.
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  java:
+    if: github.event.pull_request.merged && startsWith(github.event.pull_request.head.ref, 'release-')
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Decrypt PGP key
+        uses: IronCoreLabs/ironhide-actions/decrypt@v1
+        with:
+          keys: ${{ secrets.IRONHIDE_KEYS }}
+          input: .github/signing-key.asc.iron
+      - name: Import PGP key
+        run: gpg --batch --import .github/signing-key.asc
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Install llvm (ubuntu)
+        if: startsWith(matrix.target, 'ubuntu')
```

Comment:
We don't need this conditional since we're only running on ubuntu.
What do we need llvm for, by the way?

Reply:
```
   Compiling ironoxide-java v0.14.1 (/src/java)
The following warnings were emitted during compilation:

warning: couldn't execute `llvm-config --prefix` (error: No such file or directory (os error 2))
warning: set the LLVM_CONFIG_PATH environment variable to the full path to a valid `llvm-config` executable (including the executable itself)

error: failed to run custom build command for `ironoxide-java v0.14.1 (/src/java)`

Caused by:
  process didn't exit successfully: `/src/target/debug/build/ironoxide-java-2a3edf6451e545c1/build-script-build` (exit code: 101)
--- stdout
jni include dirs ["/root/.sdkman/candidates/java/current/include", "/root/.sdkman/candidates/java/current/include/linux"]
cargo:rerun-if-changed=/root/.sdkman/candidates/java/current/include/jni.h
cargo:warning=couldn't execute `llvm-config --prefix` (error: No such file or directory (os error 2))
cargo:warning=set the LLVM_CONFIG_PATH environment variable to the full path to a valid `llvm-config` executable (including the executable itself)
```

I'll get rid of the conditional, though. Good point.